backend/Origam.Workbench/Commands/MenuItemBuilders.cs

```diff
@@ -382,7 +380,90 @@ private void AddExtensionsToList(ArrayList list, ArrayList extensions)
 		}
 	}
 
-	public class SchemaItemEditorNamesBuilder : ISubmenuBuilder
+    public class EditItemMenuBuilder : ISubmenuBuilder
+    {
+        private WorkbenchSchemaService _schema;
+        public EditItemMenuBuilder(WorkbenchSchemaService schema) 
+        {
+            _schema = schema;
+        }
+
+        object _owner = null;
+
+        #region ISubmenuBuilder Members
+        public bool LateBound
+        {
+            get
+            {
+                return true;
+            }
+        }
+        public bool HasItems()
+        {
+            if (_schema.IsSchemaLoaded)
+            {
+                return _schema.CanEditItem(_schema.ActiveNode);
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public AsMenuCommand[] BuildSubmenu(object owner)
+        {
+            _owner = owner ?? _schema.ActiveNode;
+            var list = new List<AsMenuCommand>();
+            CreateMenuItem(list, "Item", new EditActiveSchemaItem(_schema), null);
```

Comment:
I am not sure about the naming of these menu items. `Item` does not say anything about what to expect and `GridEditor` neither, as it is too technical. For example `Property Grid` seems more understandable.